ksp_plugin_adapter/orbit_analyser.cs

```diff
@@ -403,19 +414,19 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
                                      CelestialBody primary) {
     using (new UnityEngine.GUILayout.HorizontalScope()) {
       UnityEngine.GUILayout.Label(
-          $@"Recurrence: [{
+          $@"{Localizer.Format("#Principia_OBTAnalyser_StaticText_Recurrence")} [{
```

Comment:
`Principia_OrbitAnalyser_Recurrence_CapderouTriple`; make the triple a placeholder (`Recurrence: <<1>>`), and rewrite the call as follows:
```csharp
      string νₒ = recurrence?.nuo.ToString() ?? em_dash;
      string Dᴛₒ = recurrence?.dto.ToString("+0;-0") ?? em_dash;
      string Cᴛₒ = recurrence?.cto.ToString() ?? em_dash;
      UnityEngine.GUILayout.Label(
          Localizer.Format("#Principia_OrbitAnalyser_Recurrence_CapderouTriple",
                           $"[{νₒ}; {Dᴛₒ}; {Cᴛₒ}]"),
          GUILayoutWidth(8));
```